docs/policies.md

```diff
@@ -97,7 +97,13 @@ Catalyst provides a healthcare reimbursement of **$150 a month** to members and
 
 **Provided to:** Employee-Members only
 
-Catalyst employees can contribute to a tax-deferred [Vanguard SIMPLE IRA](https://investor.vanguard.com/small-business-retirement-plans/simple-ira) and the co-op will match 3% of the employee’s wages. The IRS limit on annual employee contributions as of 2022 is $14,000. Here is a good summary of [how a SIMPLE IRA works](https://www.paychex.com/articles/employee-benefits/what-is-a-simple-ira) . Employees can contribute to the SIMPLE IRA and still contribute to a traditional individual IRA or Roth IRA.
+Catalyst employees can contribute to a [Vangaurd Solo 401(k) (also known as an Individual 401k or i401(k))](https://investor.vanguard.com/accounts-plans/small-business-retirement-plans/individual-solo-401k) and the co-op will match 10% of the employee’s wages. Catalyst members are eligible for this type of retirement account because all employees are also owners.
+
+Employees can defer 100% of their compensation up to $22,500 for 2023 ($30,000 for employees age 50 or older). Employers can contribute up to 25% of compensation not to exceed $66,000 for the 2023 tax year. **Employees can make pre-tax or Roth (after-tax) contributions to the 401(k)**. However, all **employer** contributions are made pre-tax.
+
+Employees can contribute to the Solo 401(k) and still contribute to a traditional individual IRA or Roth IRA.
+
+The members can use the high employer contribution limit of 25% to give themselves tax efficient raises and cost of living adjustments.
```

Comment:
Is this the same 25% as mentioned above?

> Employers can contribute up to 25% of compensation not to exceed $66,000 for the 2023 tax year

Also the wording "the members" is a little confusing here. Do you mean what we call member/employees?

Reply:
Yes, this is the same 25% as mentioned above. 

What word would be less confusing? Maybe: _"The members of the cooperative can decide to give themselves tax-efficient raises and cost-of-living adjustments."_

Reply:
Oh I think it was just confusing that in one sentence you used "employees" and in the next used "members". It makes it sound like two different groups of people when I think they are referring to the same. I would just pick one or the other.

Reply:
Also I would condense the references to the 25% to the same sentence/paragraph so it's more clear that it's the same.

Reply:
I reworked the last sentence to include an example of how we would use the 401k as a COLA. I want to keep the two paragraphs separate because the first one at line 102 just describes the rules as per the IRS which is why I use employee and employer terminology. I think most new folks will be more familiar with thinking about retirement plans using this language. 

The last paragraph uses more coop language because we are using this retirement account in a non-traditional manner.

Is this change better or more confusing?

Reply:
Just read through it, and it looks good!